spec/spec_helper.rb

```diff
@@ -5,6 +5,9 @@
 
 ENV["RAILS_ENV"] = "test"
 
+require "dotenv"
+Dotenv.overload(".sample.env")
```

Comment:
Ok, I read the tutorial, it's so we don't use the config setup in the container.

Don't think we need to require `dotenv` though:

```bash
λ  Hours  development ~ rails c test
[1] pry(main)> Rails.env.test?
=> true
[2] pry(main)> Dotenv
=> Dotenv
```